### ISSUE TITLE
GivePlayerMoney not working in OnPlayerSpawn?

### ISSUE THREAD

**Issue by rt-2** [NexiusTailer/Nex-AC#33]
Hello, I made this gamemode to test:
```
//
// Nex-Ac Test Script #3 (GivePlayerMoney not working)
//      by: rt-2
//      date: 2017-05-06
//
//This script will fail to give you your money when spawning (using GivePlayerMoney)
//

// Main Libraries
#include <a_samp>
#include <crashdetect>

//Anticheat
#include <nex-ac>

public OnGameModeInit()
{

	SetGameModeText("Test Script");
	AddPlayerClass(random(100), 1757.00, -1946.00, 13.56, 360.00, 0, 0, 0, 0, 0, 0);


	return 1;
}

public OnPlayerSpawn(playerid)
{
	GivePlayerMoney(playerid, 1337);
	return 1;
}

main()
{
	print("\n----------------------------------");
	printf(" 'Nex-Ac Test Script' by rt-2 ");
	print("----------------------------------\n");
}
```
And the player receive the money briefly, but then it goes away, unfortunately I do not have time to investigate this, as I am currently investigating a possible bug with tuning.
Tell me if you have the bug also?
Thank you,
rt-2

**Comment by NexiusTailer** [comment]
Yes, I also reproduced this problem on the test. Well, after researching this, the problem seems to be just in the logic of anti-money hack: As it is associated with anti-weapon cheats (purchase in ammunition) and tuning (purchases in tuning garages), so then there is the need for it to be performed in OnPlayerUpdate.
But if this anti-cheat was would in the timer, the money most likely would have time to synchronize and new amount would have time to apply to the client before it is overwritten by the previous one...

So I'm still thinking about the best solution to this problem, but for now, I advise to use your own money system on the server (which also gives anti-money hack)

**Comment by rt-2** [comment]
Well good luck,
Thank you,
rt-2

**Comment by rt-2** [comment]
This bug is still active isn't it?

**Comment by NexiusTailer** [comment]
Actually isn't :)
Sorry for closing this issue before new version release, but this bug fixed in v1.9.27 so you can test it even just now.

**Comment by rt-2** [comment]
Alrighty will test that too
Thank you,
rt-2
